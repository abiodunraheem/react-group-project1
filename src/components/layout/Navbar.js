import React from 'react';

const Navbar = () => (
  <nav className="nav-bar">
    <div className="header">
      <img src="" alt="" />
      <h1>Space Travellers Hub</h1>
    </div>
  </nav>
);

export default Navbar;
