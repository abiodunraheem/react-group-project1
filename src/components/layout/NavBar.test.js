import React from 'react';
import renderer from 'react-test-renderer';
import { HashRouter as Router } from 'react-router-dom';
import Navbar from './Navbar';

