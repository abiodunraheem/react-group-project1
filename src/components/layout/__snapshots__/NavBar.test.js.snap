// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for the navigation bar section It should render the navigation bar 1`] = `
<nav
  className="navbar"
>
  <div
    className="header"
  >
    <a
      aria-current="page"
      className="active"
      href="#/"
      onClick={[Function]}
    >
      <img
        alt="log"
        className="logo"
        src="planet.png"
      />
    </a>
    <h1>
      Space Traveller
      <span
        className="before"
      >
        s
      </span>
       
      Hub
    </h1>
  </div>
  <ul
    className="links_container"
  >
    <li>
      <a
        className=""
        href="#/rocket"
        onClick={[Function]}
      >
        Rocket
      </a>
    </li>
    <li>
      <a
        className=""
        href="#/mission"
        onClick={[Function]}
      >
        Mission
      </a>
    </li>
    <li>
      <a
        className=""
        href="#/profile"
        onClick={[Function]}
      >
        My profile
      </a>
    </li>
  </ul>
</nav>
`;
